package.json

```diff
@@ -17,7 +17,7 @@
         "----------------------TEST----------------------": "",
         "test:unit": "turbo run test --filter=./packages/*",
         "test:types": "turbo run test:types --filter=./packages/*",
-        "test:lint": "eslint --ignore-path .gitignore --max-warnings 0 packages/**/*.{js,ts,tsx,jsx}",
+        "test:lint": "eslint --ignore-path .gitignore --max-warnings 0 packages/**/*.{ts,tsx}",
```

Comment:
Whats exactly the error? I haven't seen one like that myself.
Also there are some `.js` files and there will probably be more.

Reply:
<img width="1269" alt="solid-devtools 2024-04-01 12-36-59" src="https://github.com/thetarnav/solid-devtools/assets/13445/cda7adaf-7ae3-48a5-bfcf-4cc41ef325ca">

Reply:
Can you try surrounding the original pattern with `\"`?
This has to be some os issue, because I don't have that behavior.

Reply:
yup. it does the trick. good catch.
should I wrap all other globs in quotes as well?

Reply:
Is is necessary anywhere else? It might be eslint specific.